Sources/LocalNetworkConnectivity/VLCNetworkLoginViewController.m

```diff
@@ -166,6 +161,7 @@ - (void)themeDidChange
     self.view.backgroundColor = PresentationTheme.current.colors.background;
     self.tableView.backgroundColor = PresentationTheme.current.colors.background;
     self.tableView.separatorColor = PresentationTheme.current.colors.background;
+    [self setNeedsStatusBarAppearanceUpdate];
```

Comment:
remember comment about `preferredStatusBarStyle` dependency on theme - here we also have it, but do not update it with theme change